main.js

```diff
@@ -0,0 +1,14 @@
+let $startbutton = document.querySelector('button')
+let $changeddigit = document.querySelector('p')
+
+let x = function update() {
```

Comment:
Avoid using single-letter variable names. You named your function, just use that.

```js
function update() {

}

function showNumbber() {
  setInterval(update, 1000)
}

$startButton.addEventListener('click', showNumber)
```